<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainLoja</class>
 <widget class="QMainWindow" name="MainLoja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainLoja</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1013</width>
      <height>628</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="text">
        <string>Livros</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="text">
        <string>CDs</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_4">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="text">
        <string>DVDs</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QTableWidget" name="tabela_livros">
       <property name="minimumSize">
        <size>
         <width>333</width>
         <height>600</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>333</width>
         <height>600</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Nome</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Preço</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Autor</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTableWidget" name="tabela_cds">
       <property name="minimumSize">
        <size>
         <width>333</width>
         <height>600</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>333</width>
         <height>600</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Nome</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Preço</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Num de Faixas</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QTableWidget" name="tabela_dvds">
       <property name="minimumSize">
        <size>
         <width>333</width>
         <height>600</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>333</width>
         <height>600</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Nome</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Preço</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Duração</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionLer"/>
    <addaction name="separator"/>
    <addaction name="actionSalvar"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuItem">
    <property name="title">
     <string>Item</string>
    </property>
    <addaction name="actionIncluir_Livro"/>
    <addaction name="separator"/>
    <addaction name="actionIncluir_CD"/>
    <addaction name="separator"/>
    <addaction name="actionIncluir_DVD"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuItem"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionLer">
   <property name="text">
    <string>Ler</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionIncluir_Livro">
   <property name="text">
    <string>Incluir Livro</string>
   </property>
  </action>
  <action name="actionIncluir_CD">
   <property name="text">
    <string>Incluir CD</string>
   </property>
  </action>
  <action name="actionIncluir_DVD">
   <property name="text">
    <string>Incluir DVD</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
